pyrgg/params.py

```diff
@@ -133,6 +133,10 @@
 
 PYRGG_CONFIG_SAVE_ERROR_MESSAGE = "[Error] Failed to save config file!"
 
+PYRGG_CONFIG_LIST_MESSAGE = "Config files detected in current directory are listed below :"
```

Comment:
`Config files detected in the current directory are listed below:`

Reply:
fixed in https://github.com/sepandhaghighi/pyrgg/pull/170/commits/ce9d852ec1025fb4dfaf9fa784824e708712a389